plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/adapter/controllers/AbstractCDKLaunchController.java

```diff
@@ -82,16 +82,28 @@ public abstract void launch(ILaunchConfiguration configuration, String mode,
 
 	
 	protected IProcess addProcessToLaunch(Process p, ILaunch launch, IServer s) {
+		return addProcessToLaunch(p, launch, s, true);
+	}
+	
+	protected IProcess addProcessToLaunch(Process p, ILaunch launch, IServer s, boolean terminal) {
+
 		Map<String, String> processAttributes = new HashMap<String, String>();
 		String vagrantcmdloc = VagrantBinaryUtility.getVagrantLocation(s);
 		String progName = new Path(vagrantcmdloc).lastSegment();
-		launch.setAttribute(DebugPlugin.ATTR_CAPTURE_OUTPUT, "false");
+		if( terminal ) {
+			launch.setAttribute(DebugPlugin.ATTR_CAPTURE_OUTPUT, "false");
+		}
 		processAttributes.put(IProcess.ATTR_PROCESS_TYPE, progName);
-		IProcess process = new RuntimeProcess(launch, p, vagrantcmdloc, processAttributes) {
-			protected IStreamsProxy createStreamsProxy() {
-				return null;
-			}
-		};
+		IProcess process = null;
+		if( terminal ) {
+			process = new RuntimeProcess(launch, p, vagrantcmdloc, processAttributes) {
+                protected IStreamsProxy createStreamsProxy() {
```

Comment:
any chance that you'd correct the identation here since it helps reading the code